boards/xtensa/esp32_ethernet_kit/doc/index.rst

```diff
@@ -0,0 +1,544 @@
+       using RESET_N, the oscillator module should be disabled by default and
+       turned on by ESP32 after power-up. For a reference design please see
+       `ESP32-Ethernet-Kit V1.2 Ethernet board (A) Schematic <https://dl.espressif.com/dl/schematics/SCH_ESP32-Ethernet-Kit_A_V1.2_20200528.pdf>`_.
+
+    2. The ESP32 pins GPIO16 and GPIO17 are not broken out to the
+       ESP32-WROVER-E module and therefore not available for use. If you need
+       to use these pins, please solder a module without PSRAM memory inside,
+       e.g. the ESP32-WROOM-32D or ESP32-SOLO-1.
+
+System requirements
+-------------------
+
+Prerequisites
+-------------
+
+Espressif HAL requires WiFi and Bluetooth binary blobs in order work. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west blobs fetch hal_espressif
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+Building & Flashing
+-------------------
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32_ethernet_kit
+   :goals: build
+
+The usual ``flash`` target will work with the ``esp32_ethernet_kit`` board
+configuration. Here is an example for the :ref:`hello_world`
+application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32_ethernet_kit
+   :goals: flash
+
+Open the serial monitor using the following command:
+
+.. code-block:: shell
+
+   west espressif monitor
+
+After the board has automatically reset and booted, you should see the following
+message in the monitor:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
+   Hello World! esp32_ethernet_kit
+
+Debugging
+---------
+
+As with much custom hardware, the ESP32 modules require patches to
+OpenOCD that are not upstreamed yet. Espressif maintains their own fork of
+the project. The custom OpenOCD can be obtained at `OpenOCD ESP32`_
+
+The Zephyr SDK uses a bundled version of OpenOCD by default. You can overwrite that behavior by adding the
+``-DOPENOCD=<path/to/bin/openocd> -DOPENOCD_DEFAULT_PATH=<path/to/openocd/share/openocd/scripts>``
+parameter when building.
+
+Here is an example for building the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32_ethernet_kit
+   :goals: build flash
+   :gen-args: -DOPENOCD=<path/to/bin/openocd> -DOPENOCD_DEFAULT_PATH=<path/to/openocd/share/openocd/scripts>
+
+You can debug an application in the usual way. Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32_ethernet_kit
+   :goals: debug
+
+
+Enabling Ethernet
+-----------------
+
+Enable Ethernet MAC, PHY and MDIO; add these to your device tree overlay::
```

Comment:
Nitpick: duplicated colon

Reply:
I changed this but will have to change it back...
`::` is used for code blocks, it renders as a single `:`
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks